src/main/java/uk/gov/hmcts/auth/provider/service/api/config/JwtConfiguration.java

```diff
@@ -11,20 +13,24 @@
 import java.time.Clock;
 
 @Configuration
+@EnableConfigurationProperties(AuthProviderProperties.class)
 public class JwtConfiguration {
 
+    @Autowired
+    private AuthProviderProperties providerProperties;
```

Comment:
provider == this app, right?
Perhaps just `properties`?
Or `appProperties`.

Reply:
Didn't want to introduce confusion in super simplifying the class name since it is **public**. If you think that's fine - no probs in renaming 😉